eth/tracers/api_test.go

```diff
@@ -186,7 +197,7 @@ func (b *testBackend) BlockByNumber(ctx context.Context, number rpc.BlockNumber)
 func (b *testBackend) GetTransaction(ctx context.Context, txHash common.Hash) (*types.Transaction, common.Hash, uint64, uint64, error) {
 	tx, hash, blockNumber, index := rawdb.ReadTransaction(b.chaindb, txHash)
 	if tx == nil {
-		return nil, common.Hash{}, 0, 0, errTransactionNotFound
+		return nil, common.Hash{}, 0, 0, nil
 	}
 	return tx, hash, blockNumber, index, nil
 }
```

Comment:
I had to modify the pre-existing test implementation of `GetTransaction` because it returned an error when `tx == nil`, which is different from the actual implementation (linked to above).

Either way, making this change doesn't affect any of the upstream tests.